src/com/oltpbenchmark/schema/TableSchema.java

```diff
@@ -0,0 +1,60 @@
+package com.oltpbenchmark.schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TableSchema {
+    private String name;
+    private final List<Column> columns = new ArrayList<>();
+    private String primaryKey = null;
+    private String partitionKey = null;
+
+    public String name() { return name; } 
+
+    static class Builder {
```

Comment:
This version of builder seems very strange because calling of `build` function multiple times always returns same object.

the following common scenario for builders will produce unexpected results

```
Builder b = new Builder();
TableSchema s1 = b.build();
b.column("col1", "int");
TableSchema s2 = b.build();
```

Reply:
I looked up another example on Builders and now changed the way the builder works, let me know if this is better. The new Builder always returns a different Schema object.